crates/turborepo-lib/src/commands/prune.rs

```diff
@@ -87,7 +89,9 @@ pub async fn prune(
     output_dir: &str,
     telemetry: CommandEventBuilder,
 ) -> Result<(), Error> {
-    telemetry.track_prune_method(docker);
+    telemetry.track_prune_arg_method(docker);
+    telemetry.track_arg_usage("out-dir", output_dir != DEFAULT_OUTPUT_DIR);
```

Comment:
So if a user says `--out-dir=out` , we would get false for this? Could we distinguish the default being provided, vs turbo doing the default thing? I wouldn't care if we were sending real values, but if our telemetry is tracking "set" vs "default", this implementation could be a bit misleading

Reply:
Yes that's correct, and that's a good call. I'm not sure if clap lets us differentiate between that. But I can check on it.

Reply:
It doesn't. If we really want to do this we would need to change the type to be `Option<String>` and defer using the default until time of use.